changelog.md

```diff
@@ -0,0 +1,22 @@
+**Beta features**
+1. Multiple sidechain nodes network.
+2. Mainchain synchronization: Cross-chain Transfer Protocol support for sidechain declaration: forward and backward transfers.
+3. Proof-of-Stake consensus protocol for sidechain based on Ouroboros Praos consensus protocol.
```

Comment:
Let's specify the name of the protocol like `Latus - Proof-of-Stake ....` and put the link to whitepaper 2.0

Reply:
done